cli/bundle/bundle_test.go

```diff
@@ -1 +1,33 @@
 package bundle
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+func TestBundle(t *testing.T) {
+	gopath := os.Getenv("GOPATH")
+	cfssl := path.Join(gopath, "bin", "cfssl")
+	testdata := path.Join(gopath, "src", "github.com", "cloudflare", "cfssl", "bundler", "testdata")
+	pkg := path.Join(gopath, "src", "github.com", "cfssl_trust")
+	//cfssl bundle [-ca-bundle file] [-int-bundle file] [-key keyfile] [-flavor int] [-metadata file] CERT
+	out1, err1 := exec.Command(cfssl, "bundle",
+		"-cert="+path.Join(testdata, "bunnings.pem"),
+		"-ca-bundle="+path.Join(pkg, "ca-bundle.crt"),
+		"-int-bundle="+path.Join(pkg, "int-bundle.crt"),
+		"-metadata="+path.Join(pkg, "ca-bundle.crt.metadata")).CombinedOutput()
+	fmt.Printf("cfssl bundle [-ca-bundle file] [-int-bundle file] [-metadata file] [-key keyfile] [-flavor int]  CERT\n%s\n", string(out1))
+	fmt.Println("Error: ", err1)
```

Comment:
printing out result is not a real test, since most of time stdout/stderr output is omitted. It's expected we do t.Error(), t.Fail() or t.Fatal() at least once.